### PR TITLE
[Feature] Introduce CDC job submission form

### PR DESCRIPTION
**Pull request by hdygxsj**
### Purpose
Add a form for submitting Cdc action jobs to the Flink session cluster.
image

image

### Api
http://127.0.0.1:10088/api/cdc-job-definition/${jobId}/submit

### CODE REVIEW

File: paimon-web-ui/src/form-lib/cdc/use-paimon.ts [hunk 1]
```diff
@@ -15,109 +15,110 @@ KIND, either express or implied.  See the License for the
...
+  const data = item.data
+  const model = reactive({
+    catalog: data.catalog || '',
+    other_configs: data.other_configs || '',
+    database: data.database || '',
+    table_name: data.table_name || '',
+    primary_key: data.primary_key || '',
+    partition_column: data.partition_column || '',
+    other_configs2: data.other_configs2 || ''
+  })
```
Comment (zhangmo8) [inline paimon-web-ui/src/form-lib/cdc/use-paimon.ts:41]: You can assign values using extenders, but pay attention to the type
```suggestion
  const model = reactive({ ...data })
```

File: paimon-web-ui/src/locales/zh/modules/cdc.ts [hunk 2]
```diff
@@ -49,4 +49,5 @@ export default {
   synchronization_configuration: '同步配置',
   primary_key: '主键',
   partition_column: '分区列',
+  submit_cdc_job:'提交CDC作业'
```
Comment (zhangmo8) [inline paimon-web-ui/src/locales/zh/modules/cdc.ts:52]:
```suggestion
  submit_cdc_job:'提交 CDC 作业'
```

File: paimon-web-ui/src/views/cdc/components/list/index.tsx [hunk 3]
```diff
@@ -20,10 +20,11 @@ import { useTable } from './use-table'
 
 export default defineComponent({
   name: 'ListPage',
-  setup() {
+  emits:['cdcJobSubmit'],
+  setup(props,ctx) {
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/cdc/components/list/index.tsx:24]: If you do not use this props, please use an '_' as the parameter name. 
```suggestion
  setup(_, ctx) {
```

File: paimon-web-ui/src/views/cdc/components/list/use-table.ts [hunk 4]
```diff
@@ -15,106 +15,113 @@ KIND, either express or implied.  See the License for the
...
-                title: t('cdc.create_user'),
-                key: 'createUser',
-                resizable: true
-            },
-            {
-                title: t('cdc.create_time'),
-                key: 'createTime',
-                resizable: true
+import { useCDCStore } from '@/store/cdc'
+import type { Router } from 'vue-router'
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/cdc/components/list/use-table.ts:19]: sort the import. Of course, we will fix it later in the process of eslint improvement. This is just a reminder.

File: paimon-web-ui/src/views/cdc/components/list/use-table.ts [hunk 5]
```diff
@@ -15,106 +15,113 @@ KIND, either express or implied.  See the License for the
...
+      {
+        title: t('cdc.synchronization_type'),
+        key: 'cdcType',
+        resizable: true,
+        render: (row: any) => {
+          if (row.cdcType == 0) {
+            return t('cdc.single_table_synchronization')
+          } else if (row.cdcType == 1) {
+            return t('cdc.whole_database_synchronization')
+          }
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/cdc/components/list/use-table.ts:41]: The number type can be judged as a Boolean

```suggestion
    const message = row.cdcType ? 'cdc.whole_database_synchronization' : 'cdc.single_table_synchronization'
    return t(message)
```

**Review by zhangmo8 — APPROVED** [review]
LGTM

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**